src/main/java/wooteco/subway/member/ui/MemberController.java

```diff
@@ -27,47 +27,47 @@ public MemberController(MemberService memberService) {
         this.memberService = memberService;
     }
 
-    @PostMapping("/members")
+    @PostMapping
     public ResponseEntity<Void> createMember(@RequestBody @Valid MemberRequest request) {
         MemberResponse member = memberService.createMember(request);
         return ResponseEntity.created(URI.create("/members/" + member.getId())).build();
     }
 
-    @GetMapping("/members/{id}")
+    @GetMapping("/{id}")
     public ResponseEntity<MemberResponse> findMember(@PathVariable Long id) {
-        MemberResponse member = memberService.findMember(id);
+        MemberResponse member = memberService.findById(id);
         return ResponseEntity.ok(member);
     }
 
-    @PutMapping("/members/{id}")
+    @PutMapping("/{id}")
     public ResponseEntity<MemberResponse> updateMember(@PathVariable Long id,
```

Comment:
```suggestion
    public ResponseEntity<Void> updateMember(@PathVariable Long id,
```

<img width="668" alt="스크린샷 2021-05-16 오후 1 46 14" src="https://user-images.githubusercontent.com/18049492/118385796-1d956880-b64d-11eb-8ba8-11b75a713fc9.png">

`ResponseEntity`의 타입 파라미터는 "the body type"을 나타낸답니다. 따라서 응답의 바디가 비어 있는 경우에는 `Void` 타입으로 지정해 주시면 돼요. 전반적으로 점검해 볼까요?